Permanent/ViewControllers/LegacyPlanning/ViewController/LegacyPlanningStewardViewController.swift

```diff
@@ -86,7 +86,9 @@ class LegacyPlanningStewardViewController: BaseViewController<LegacyPlanningView
         overlayView.alpha = 0
         
         titleLabelSetup()
-        backButtonSetup()
+        if let viewControllersCount = navigationController?.viewControllers.count, viewControllersCount > 1 {
```

Comment:
wouldn't it work like
if navigationController?.viewControllers.count > 1 {
backButtonSetup()
}